flaml/tune/searcher/flow2.py

```diff
@@ -68,6 +70,19 @@ def __init__(
             resource_multiple_factor: A float of the multiplicative factor
                 used for increasing resource.
             cost_attr: A string of the attribute used for cost.
+            lexico_objectives: A dictionary with four elements.
```

Comment:
Should revise.

Reply:
This is a old version. We have revised now